app/service/rest.py

```diff
@@ -262,10 +264,11 @@ def update_service(service_id):
     if 'letter_branding' in req_json:
         letter_branding_id = req_json['letter_branding']
         service.letter_branding = None if not letter_branding_id else LetterBranding.query.get(letter_branding_id)
-    dao_update_service(service)
 
-    if 'default_email_is_french' in req_json:
-        service.default_email_is_french = req_json['default_email_is_french']
```

Comment:
This piece was not needed. A test proves that it still works https://github.com/cds-snc/notification-api/blob/c32ec8ea4aacf51d87da41031031f543d96261e9/tests/app/service/test_rest.py#L712-L719